### ISSUE TITLE
Feral's Primal Wrath not counted as a bleed

### ISSUE THREAD

**Issue by Anatta336** [simulationcraft/simc#4744]
### Describe the bug
The feral druid talent Primal Wrath doesn't count as a "bleeding" effect on targets.

### Expected behavior
When using Primal Wrath with no other bleed effects the target should get the Bleeding debuff.
APL checks for `debuff.bleeding.up` should return true when Primal Wrath is active.
The abilities Shred and Swipe should do 20% additional damage against targets bleeding with Primal Wrath (as they do against targets bleeding from anything else.) In-game testing confirms that Primal Wrath's bleed does count as a bleed for increasing Shred and Swipe damage.

### To Reproduce
simc-810-02-win64-4e56b8cbc0 (a recent nightly build)
```
druid="Anatta"
spec=feral
level=120
race=tauren
role=attack
talents=0000030

actions.precombat=cat_form
actions.precombat+=/snapshot_stats

actions=auto_attack

# Use Primal Wrath at 5 combo
actions+=/pool_resource,for_next=1
actions+=/primal_wrath,if=combo_points>=5

# shred when there's no bleed active
actions+=/pool_resource,for_next=1
actions+=/shred,if=debuff.bleeding.down

# swipe when there's a bleed
actions+=/pool_resource,for_next=1
actions+=/swipe_cat,if=debuff.bleeding.up
```
This APL will have a good uptime of Primal Wrath's DoT but never casts Swipe, and Shred will always do damage as if there is no bleed active.

**Comment by scamille** [comment]
This still seems relevant for shadowlands @simulationcraft/druid and can be replicated with the given profile and ```optimal_raid=0```

**Comment by gastank** [comment]
Should be fixed in c1ae91fa9c250dc86fdf106d879f99dbf924e15b
